lib/capybara-screenshot/cucumber.rb

```diff
@@ -18,8 +19,10 @@
         require "base64"
         #encode the image into it's base64 representation
         image = open(saver.screenshot_path, 'rb') {|io|io.read}
-        encoded_img = Base64.encode64(image)
+        #print image to screen, if imgcat is available
+        system("imgcat #{saver.screenshot_path}") if find_executable 'imgcat'
```

Comment:
Yep :+1: It might also be a good idea to abstract this out, so it can be used in the rspec and minitest classes.